datafusion/functions/src/crypto/basic.rs

```diff
@@ -198,25 +167,7 @@ fn hex_encode<T: AsRef<[u8]>>(data: T) -> String {
     }
     s
 }
-pub fn utf8_or_binary_to_binary_type(
```

Comment:
Refactored away, was only used for return types in sha/md5/digest (which are now simplified)